src/test/java/com/jcabi/aether/MavenClasspathTest.java

```diff
@@ -55,35 +56,76 @@
 public final class MavenClasspathTest {
 
     /**
+     * Group of test artifact.
+     */
+    private static final String GROUP = "junit";
```

Comment:
@mkordas "test data" - its just a constant to avoid checktyle complaining about duplicate strings...